components/ActionSheet/README.md

```diff
@@ -0,0 +1,111 @@
+<!-- This file was auto-generated using ./scripts/generate_readme ActionSheet -->
+
+# Action Sheet
+
+[![Open bugs badge](https://img.shields.io/badge/dynamic/json.svg?label=open%20bugs&url=https%3A%2F%2Fapi.github.com%2Fsearch%2Fissues%3Fq%3Dis%253Aopen%2Blabel%253Atype%253ABug%2Blabel%253A%255BActionSheet%255D&query=%24.total_count)](https://github.com/material-components/material-components-ios/issues?q=is%3Aopen+is%3Aissue+label%3Atype%3ABug+label%3A%5BActionSheet%5D)
+
+Material design action sheets should be used as an overflow menu. An Action Sheet comes up from the bottom of
+the screen and displays actions a user can take.
+
+<img src="docs/assets/actionSheetPortrait.gif" alt="An animation showing a Material Design Action Sheet." width="150"> <img src="docs/assets/actionSheetLandscape.gif" alt="An animation showing a Material Design Action Sheet." width="324">
```

Comment:
For readme docs we prefer to use device recordings so that the mouse and device chrome aren't shown. You can make a recording in QuickTime by installing the app on your device, connecting it to your computer, opening QuickTime, starting a new movie recording, and selecting your device as the camera.

Reply:
Done